### PR TITLE
Template update for nf-core/tools v3.1.0

### PR DESCRIPTION
**Pull request by sateeshperi**

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -164,40 +163,37 @@ nextflow pull nf-core/methylseq
...
 
-:::info
-We highly recommend the use of Docker or Singularity containers for full pipeline reproducibility, however when this is not possible, Conda is also supported.
-:::
+> [!IMPORTANT]
+> We highly recommend the use of Docker or Singularity containers for full pipeline reproducibility, however when this is not possible, Conda is also supported.
 
-The pipeline also dynamically loads configurations from [https://github.com/nf-core/configs](https://github.com/nf-core/configs) when it runs, making multiple config profiles for various institutional clusters available at run time. For more information and to see if your system is available in these configs please see the [nf-core/configs documentation](https://github.com/nf-core/configs#documentation).
+The pipeline also dynamically loads configurations from [https://github.com/nf-core/configs](https://github.com/nf-core/configs) when it runs, making multiple config profiles for various institutional clusters available at run time. For more information and to check if your system is suported, please see the [nf-core/configs documentation](https://github.com/nf-core/configs#documentation).
```
Comment (jfy133) [inline docs/usage.md:191]:
```suggestion
The pipeline also dynamically loads configurations from [https://github.com/nf-core/configs](https://github.com/nf-core/configs) when it runs, making multiple config profiles for various institutional clusters available at run time. For more information and to check if your system is supported, please see the [nf-core/configs documentation](https://github.com/nf-core/configs#documentation).
```

File: nextflow.config [hunk 2]
```diff
@@ -274,30 +271,95 @@ set -C # No clobber - prevent output redirection from overwriting files.
...
     name            = 'nf-core/methylseq'
-    author          = """Phil Ewels, Sateesh Peri"""
+    contributors    = [
+        [
+            name: 'Phil Ewels',
+            affiliation: '',
+            email: '',
+            github: 'https://github.com/ewels',
+            contribution: ['author', 'contributor'], // List of contribution types ('author', 'maintainer' or 'contributor')
+            orcid: ''
```
Comment (jfy133) [inline nextflow.config:300]: You can get half these ORCIDs and affiliations from the two nf-core papers I think

**Review by jfy133 — APPROVED** [review]
Comments for completeness, but otherwise no blockers here!

File: nextflow.config (hunk 2 above)
Comment (sateeshperi) [reply nextflow.config:300]: will do

**Pull request merged**